app/views/shared/_resource_details.html.erb

```diff
@@ -84,7 +84,7 @@
           <%= t(".labels.contact_email") %>
         </h3>
         <p>
-          <a href="mailto: <%= resource.contact_email %>">
+          <a href="mailto:<%= resource.contact_email %>">
```

Comment:
W3C error about the `href` mailto having an empty space (was also causing the link to return a 500 error)